integration-tests/wiremock-tests/notifcation-service.test.js

```diff
@@ -1,27 +1,65 @@
-const { stubNotificationGet, stubNotificationUpdate } = require('../mockApis/notificationService')
+const {
+  stubNotificationCount,
+  stubNotificationPreferencesGet,
+  stubNotificationUpdate,
+} = require('../mockApis/notificationService')
 const { resetStubs } = require('../mockApis/wiremock')
-const notifcationService = require('../../server/services/notificationService')
+const notifciationService = require('../../server/services/notificationService')
+
+const fakeShiftNotifications = [
+  {
+    description: 'Your activity on Wednesday 29 May 2019 at 17:00 - 17:30 has changed to [Paternity Leave].',
+    shiftModified: '2019-05-29T13:53:01.000Z',
+    processed: true,
+  },
+  {
+    description: 'Your activity on Wednesday 29 May 2019 at 15:00 - 17:00 has changed to [FMI Training].',
+    shiftModified: '2019-05-29T13:46:19.000Z',
+    processed: true,
+  },
+  {
+    description: 'Your activity on Wednesday 29 May 2019 at 17:00 - 17:30 has changed to [Late Roll (OSG)].',
+    shiftModified: '2019-05-29T13:43:37.000Z',
+    processed: true,
+  },
+  {
+    description: 'Your activity on Wednesday 29 May 2019 at 15:00 - 17:00 has changed to [FMI Training].',
+    shiftModified: '2019-05-29T13:36:43.000Z',
+    processed: true,
+  },
+]
 
 describe('Wiremock setup for notification service', () => {
   beforeAll(async () => {
     await resetStubs()
-    await stubNotificationGet()
+    await stubNotificationCount()
+    await stubNotificationPreferencesGet()
     await stubNotificationUpdate()
   })
 
-  test('notifcationService.get is stubbed', async () => {
+  test('notificationService.count is stubbed', async () => {
+    try {
+      const response = await notifciationService.countUnprocessedNotifications()
```

Comment:
`const { status, data } = await ...`

Reply:
See previous comment. All code in `/wiremock-tests` is up for review in my next PR.